packages/lab/src/calendar/internal/CalendarWeekHeader.tsx

```diff
@@ -33,13 +33,13 @@ export const CalendarWeekHeader = forwardRef<
       {...rest}
     >
       {weekdaysShort.map((day, index) => (
-        <small
+        <abbr
```

Comment:
Why was this change made?

Reply:
because it is an abbreviation and the tag was more meaningful, but i forgot to add title and i guess it being aria-hidden defeats the purpose, will bring `small` back